RecoMuon/MuonIdentification/python/muons1stStep_cfi.py

```diff
@@ -77,7 +77,10 @@
                                           OverlapDTheta = cms.double(0.02), # 1.14 degrees
                                           ClusterDPhi   = cms.double(0.6), # 34 degrees
                                           ClusterDTheta = cms.double(0.02) # 1.14
-    )
+    ),
+
+    # tracker muon arbitration
+    arbitrateTrackerMuons = cms.bool(True)
```

Comment:
The effect of this flag is that it ends up being set in all reconstructions (earlyMuons, a variety of cosmics etc).
Since the default is false, it stays unset in HLT, but even that is probably until the next re-parsing of confDB.